src/Symfony/Component/Mailer/Bridge/Mailjet/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+4.4.0
```

Comment:
Should be 5.1 IMO

Reply:
Okay, I did change that, thanks :)

Reply:
I get a probem on the Travis I did not have before, is it something I can fix ?

Reply:
i do not know, maybe make sure your branch is rebase on the master
and retry build?